includes/polling/ports.inc.php

```diff
@@ -620,6 +620,11 @@
             $this_port['ifVlan'] = $this_port['dot1qPvid'];
         }
 
+        // Set ifConnectorPresent to null when the device does not support IF-MIB truth values.
+        if (isset($this_port['ifConnectorPresent']) && !in_array($this_port['ifConnectorPresent'], array('true', 'false'))) {
+            $this_port['ifConnectorPresent'] = 'null';
```

Comment:
This should be null without quotes.

Reply:
also, please use short array syntax ['true', 'false']

Reply:
Done.